dashboard/src/main/kotlin/com/ritense/dashboard/autoconfigure/DashboardAutoConfiguration.kt

```diff
@@ -135,4 +149,12 @@ class DashboardAutoConfiguration {
             clearTables
         )
     }
+
+    @Bean
+    fun dashboardSpecificationFactory(
+        @Lazy dashboardService: DashboardService,
```

Comment:
I know why the `@Lazy` is here, but I'm going to check if there's another way to prevent it.